.lycheeignore

```diff
@@ -0,0 +1 @@
+^/meeting-notes/.*
```

Comment:
@keunes I don't think this works in a `.lycheeignore` file. I think we need to create a `lychee.toml` and add

```toml
exclude_path = ["meeting-notes"]
```